core/Plugin/ReportsProvider.php

```diff
@@ -117,7 +117,7 @@ private static function getMapOfModuleActionsToReport()
     public function getAllReports()
     {
         $reports = $this->getAllReportClasses();
-        $cacheId = CacheId::languageAware('Reports' . md5(implode('', $reports)));
+        $cacheId = CacheId::siteAware(CacheId::languageAware('Reports' . md5(implode('', $reports))));
```

Comment:
The method itself does not really need an site aware caching, but the event `Report.addReports` might add reports based on the site id. Which is actually the case in some of our plugins...